packages/sanity/src/core/store/bundles/reducer.ts

```diff
@@ -0,0 +1,109 @@
+import {type BundleDocument} from './types'
+
+interface BundleAddedAction {
+  payload: BundleDocument
+  type: 'BUNDLE_ADDED'
+}
+
+interface BundleDeletedAction {
+  id: string
+  type: 'BUNDLE_DELETED'
+}
+
+interface BundleUpdatedAction {
+  payload: BundleDocument
+  type: 'BUNDLE_UPDATED'
+}
+
+interface BundlesSetAction {
+  bundles: BundleDocument[]
```

Comment:
Just a nitpick, nothing important. For consistency, should this be `payload` instead?

```suggestion
  payload: BundleDocument[]
```